doc/source/data_structures.rst

```diff
@@ -0,0 +1,121 @@
+operations can be applied between two ``GeoSeries``, in which case the
+operation is carried out elementwise.  The two series will be aligned
+by matching indices.  Binary operations can also be applied to a
+single geometry, in which case the operation is carried out for each
+element of the series with that geometry.  In either case, a
+``Series`` or a ``GeoSeries`` will be returned, as appropriate.
+
+A short summary of a few attributes and methods for GeoSeries is 
+presented here, and a full list can be found in the :doc:`all attributes and methods page <reference>`. 
+There is also a family of methods for creating new shapes by expanding 
+existing shapes or applying set-theoretic operations like "union" described
+in :doc:`geometric manipulations <geometric_manipulations>`.
+
+Attributes
+^^^^^^^^^^^^^^^
+* ``area``: shape area (units of projection -- see :doc:`projections <projections>`)
+* ``bounds``: tuple of max and min coordinates on each axis for each shape
+* ``total_bounds``: tuple of max and min coordinates on each axis for entire GeoSeries
+* ``geom_type``: type of geometry. 
+* ``is_valid``: tests if coordinates make a shape that is reasonable geometric shape (`according to this <http://www.opengeospatial.org/standards/sfa>`_).
+
+Basic Methods
+^^^^^^^^^^^^^^
+
+* ``distance(other)``: returns ``Series`` with minimum distance from each entry to ``other``
+* ``centroid``: returns ``GeoSeries`` of centroids
+* ``representative_point()``:  returns ``GeoSeries``of points that are guaranteed to be within each geometry. It does **NOT** return centroids.  
+* ``to_crs()``: change coordinate reference system. See :doc:`projections <projections>`
+* ``plot()``: plot ``GeoSeries``. See :doc:`mapping <mapping>`.
+
+Relationship Tests
+^^^^^^^^^^^^^^^^^^^
+
+* ``almost_equals(other)``: is shape almost the same as ``other`` (good when floating point precision issues make shapes slightly different)
+* ``contains(other)``: is shape contained within ``other``
+* ``intersects(other)``: does shape intersect ``other``
+
+
+GeoDataFrame
+------------
+
+A ``GeoDataFrame`` is a tablular data structure that contains a ``GeoSeries``. 
+
+The most important property of a ``GeoDataFrame`` is that it always has one ``GeoSeries`` column that holds a special status. This ``GeoSeries`` is referred to as the ``GeoDataFrame``'s "geometry". When a spatial method is applied to a ``GeoDataFrame`` (or a spatial attribute like ``area`` is called), this commands will always act on the "geometry" column. 
+
+The "geometry" column -- no matter its name -- can be accessed through the ``geometry`` attribute (``gdf.geometry``), and the name of the ``geometry`` column can be found by typing ``gdf.geometry.name``.  
+
+A ``GeoDataFrame`` may also contain other ``GeoSeries``, but only one ``GeoSeries`` can be the active geometry at a time. To change which ``GeoSeries`` is the geometry, use the ``set_geometry`` method:
```

Comment:
This statement is not really correct I think. As far as I can see, a GeoDataFrame can only hold one GeoSeries. Other columns with shapely objects are just returned as normal Series objects. 

So of course you can have multiple columns with shapely.geometry values, but those aren't exactly GeoSerieses